### PR TITLE
MM-61068 - send WS event on scheduled message job send

### PR DESCRIPTION
**Pull request by pvev**
#### Summary
This PR sends a WS event when  a scheduled post is send as a post by de scheduled post job.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-61068

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by harshilsharma63** [comment]
Tried locally, works great!

**Review by larkox — APPROVED** [review]

File: server/channels/app/scheduled_post_job.go
Comment (larkox) [inline server/channels/app/scheduled_post_job.go:1]: 0/5 should we add tests to verify websocket events are being sent? (or at least that PublishScheduledPostEvent is being called)

**Review by yasserfaraazkhan — APPROVED** [review]
the failed message is marked red.
Successfull message is seen in channel and disappears from list.

**Pull request merged**